andela/kenya-rc
# #154715724 improve media gallery

#### What does this PR do?
- This PR allows a user to see a carousel on the landing page
- This PR allows a user to see a hover effect in the categories section of the homepage
- This PR allows a user to see a more attractive navigation bar on the landing page
- This PR allows a user to see a proper spacing of products in the featured product section on the landing page

Description of Task to be completed?
N/A

#### How should this be manually tested?
- Run reaction to start the server
- Go to localhost:3000 to see the landing page

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
- improved media gallery #154715724

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-02-15 at 10 32 14 am" src="https://user-images.githubusercontent.com/28443813/36249711-d5fa465c-123b-11e8-95e3-2c8fbed4064b.png">
<img width="1440" alt="screen shot 2018-02-15 at 10 32 41 am" src="https://user-images.githubusercontent.com/28443813/36249753-f692a12a-123b-11e8-9ab5-cedde52346f0.png">



#### Questions:
N/A
